org.duckstation.DuckStation.yml

```diff
@@ -0,0 +1,55 @@
+app-id: org.duckstation.DuckStation
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: duckstation-nogui
+finish-args:
+  - --device=all
+  - --filesystem=host:ro
```

Comment:
Does it respect the XDG spec to save settings and save files in a writable directory?

Reply:
Seems so: https://github.com/stenzek/duckstation/blob/56243a26ec7ceffa3a22cd9a4602d62e7330121a/src/frontend-common/common_host_interface.cpp#L935, `XDG_DATA_HOME` is used.

Reply:
Neat, thanks.